detox/android/detox/src/test/java/com/wix/invoke/JsonParserTest.java

```diff
@@ -62,6 +62,17 @@ public void fromJsonTargetInvocationEspressoDetox() {
         assertThat(parse("targetInvocationEspressoDetox.json")).isEqualToComparingFieldByFieldRecursively(perform);
     }
 
+    @Test
+    public void fromJsonTargetInvocationEspressoWebDetox() {
+//        EspressoWebDetox.getWebView().element(DetoxWebAtomMatcher.matcherForId("textInput")).tap();
```

Comment:
delete?

Reply:
I would like to keep it, it helps to better understand how the command looks like.
Also, other tests have that.